variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/factories/VariantVcfFactoryTest.java

```diff
@@ -551,32 +552,34 @@ public void testParseInfo() {
     @Test
     public void testVariantIds() {
 
+        Set<String> emptySet = new HashSet<>();
         // test that an ID is properly handled
         String line = "1\t1000\trs123\tC\tT\t.\t.\t.";
         List<Variant> expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1000, 1000, "C", "T"));
-        expResult.get(0).setIds(Collections.singleton("rs123"));
+        //EVA-942 - Since we ignore IDs submitted through VCF, they are no longer part of the expected result
+        //expResult.get(0).setIds(Collections.singleton("rs123"));
         List<Variant> result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
-        assertEquals(expResult.get(0).getIds(), result.get(0).getIds());
+        assertEquals(expResult.get(0).getIds(), emptySet);
 
         // test that the ';' is used as the ID separator (as of VCF 4.2)
         line = "1\t1000\trs123;rs456\tC\tT\t.\t.\t.";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1000, 1000, "C", "T"));
-        expResult.get(0).setIds(new HashSet<>(Arrays.asList("rs123", "rs456")));
+        //expResult.get(0).setIds(new HashSet<>(Arrays.asList("rs123", "rs456")));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
-        assertEquals(expResult.get(0).getIds(), result.get(0).getIds());
+        assertEquals(expResult.get(0).getIds(), emptySet);
 
         // test that a missing ID ('.') is not added to the IDs set
         line = "1\t1000\t.\tC\tT\t.\t.\t.";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1000, 1000, "C", "T"));
-        expResult.get(0).setIds(
-                Collections.<String>emptySet());    // note!: we store a "." as an empty set, not a set with an empty string
+        //expResult.get(0).setIds(
+                //Collections.<String>emptySet());    // note!: we store a "." as an empty set, not a set with an empty string
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
-        assertEquals(expResult.get(0).getIds().size(), result.get(0).getIds().size());
+        assertEquals(expResult.get(0).getIds().size(), emptySet.size());
```

Comment:
why the `.size()`? instead of just the set, as in the other assertions??

Reply:
I did not write this test but my understanding was that this part of the test case dealt with validating the size of the result set and not the result itself.

Reply:
oh, I might remember, maybe `Collections.<String>emptySet()` is not equals to `new HashSet<String>()`. anyway, try putting `assertEquals(expResult.get(0).getIds(), emptySet);`.

Reply:
Again, check result and not expResult